libs/components/src/next/infotext/stories/vl-infotext.stories-doc.mdx

```diff
@@ -0,0 +1,40 @@
+import { ArgTypes, Canvas } from '@storybook/addon-docs';
+import * as VlInfotextStories from './vl-infotext.stories';
+
+# Infotext - next
+
+Gebruik de `infotext-next` component om belangrijke nummers duidelijk weer te gevem.<br/>
+
+Dit component is de volgende versie van de [vl-infotext](/docs/elements-infotext--documentatie) component.
+
+## Voorbeeld
+
+```js
+import { VlInfotextComponent } from '@domg-wc/components/next/infotext';
+```
+
+```html
+<vl-infotext-next></vl-infotext-next>
+```
+
+<Canvas of={VlInfotextStories.InfotextDefault} />
+
+## Configuratie
+
+<ArgTypes of={VlInfotextStories.InfotextDefault} />
+
+## Varianten
+
+### Badge
+
+<Canvas of={VlInfotextStories.InfotextBadge} />
+
+### Link
+
+<Canvas of={VlInfotextStories.InfotextLink} />
+
+## Referenties
```

Comment:
Kris is begonnen met `## Achtergrond` te plaatsen ipv `## Referenties` in zijn `vl-properties-next` ticket. Op zich is dit niet zo heel erg verschillend van betekenis.

@kspeltix is dit iets dat we vanaf nu willen aanhouden voor de nieuwe Lit componenten?

Reply:
Na bespreking behouden we `## Referenties`.